cheatsheets/Swift.rb

```diff
@@ -28,20 +28,36 @@ class MyClass : OptionalSuperClass, OptionalProtocol {
       notes <<-'END'
         ``` swift
         class MyClass {
+            // Class Methods
+            static func course() -> String { return "Swift!" }
```

Comment:
For this to be a "Class Method" it should be `class func` not `static func`. Typically `static func` should be used for value types and `class func` should be used for reference types.

Additionally I would consider removing `return` (not just here but wherever possible) to highlight cases where it is not necessary.
```suggestion
            static func course() -> String { "Swift!" }
```